storybook/stories/components/NavInternal/NavInternalPanel.stories.js

```diff
@@ -0,0 +1,282 @@
+import { NavInternalPanelTemplate } from './NavInternalPanel'
+
+export default {
+  title: 'Global Layout/Headers/For Internal Sites/Elements/Panel',
+  excludeStories: /.*Data$/,
+  argTypes: {
+    title: {
+      type: { name: 'string', required: false },
+      description: 'The link text of the parent menu item for this panel.',
+    },
+    url: {
+      type: { name: 'string', required: false },
+      description: 'The link URL of the parent menu item for this panel.',
+    },
+    repeatInPanel: {
+      type: { name: 'boolean', required: false },
+      description:
+        'Enable to display the parent menu item as a link at the top of this panel. This may be useful because top-level menu items with panels cannot be clicked on to go to their URL; clicking only opens and closes the panel.',
+    },
+    repeatedTitle: {
+      type: { name: 'string', required: false },
+      description:
+        'When `repeatInPanel` is true, this property may be set to change the text of the parent menu item link in the panel. For example, if the parent menu item’s title is “Processes” and the panel has a subset of the most popular processes, you might want the parent menu item link to be “View all processes”',
+    },
+    highlights: {
+      type: { name: 'array', required: false },
+      description: 'This template is optimized for a maximum of 3 highlights.',
+    },
+    menuItems: {
+      type: { name: 'array', required: false },
+      description: 'Menu items, including nested menus.',
+    },
+    allowSectionBreaks: {
+      type: { name: 'boolean', required: false },
+      description:
+        'If a nav [Section](/?path=/docs/global-layout-headers-for-internal-sites-elements-section--basic-heading) can flow into the next column (only applies if `allowSectionColumns` is `true`.',
+      table: {
+        defaultValue: { summary: 'true' },
+      },
+    },
+    allowSectionColumns: {
+      type: { name: 'boolean', required: false },
+      description:
+        'If multiple columns should be used to display nav [Sections](/?path=/docs/global-layout-headers-for-internal-sites-elements-section--basic-heading). If set to `false`, all nav items will appear in a single column. Useful if you only have a couple items in the nav.',
+      table: {
+        defaultValue: { summary: 'true' },
+      },
+    },
+  },
+  parameters: {
+    viewMode: 'docs',
+    docs: {
+      description: {
+        component:
+          'The contents of a mega-menu-style dropdown. Intended for use within the [Internal Header](global-layout-headers-for-internal-sites--with-navigation)',
```

Comment:
I was struggling with this, and depending on the context, you either need to include the slash, or omit it. And in some places it's just not possible. I'll review all the ones you pointed out.

Reply:
So in my attempt to fix this, I found that links aren't working properly when setting a description using the [Description DocBlock](https://storybook.js.org/docs/ember/writing-docs/doc-block-description). Links work fine in mdx documents, however, so the fix for this would be to convert any stories that have this to an mdx document. I think just the few that you've pointed out in this review.

Reply:
Ha, right. And in doing this I remembered why I didn't do these stories (at least NavInternalPanel) in mdx to begin with. Something about the NavInternalPanel stories, perhaps the javacsript variables, causes the entire page to hang. I'll have to think on a workaround for this. I may omit the links for now, or write hardcoded ones that correspond with the production environment. Not a great solution, but better than no link?

Reply:
Yeah, that seems like a reasonable workaround to me.